regression/contracts/assigns-enforce-malloc-zero/main.c

```diff
@@ -0,0 +1,34 @@
+#include <stdlib.h>
+
+int MAX_SIZE = __CPROVER_max_malloc_size;
```

Comment:
Nit pick: I'm not even sure why this is necessary, but if it's just about having a shorter name then a `#define` would seem more idiomatic?

Reply:
I'm allergic to macros, but if its just about having a shorter name then I'd go with "const int MAX_SIZE = ...;"

Reply:
fixed